src/libs/Navigation/shouldShowDeeplink/index.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'underscore';
+import CONST from '../../../CONST';
+/**
+ * @returns {Boolean}
+ */
+export default function shouldShowDeeplink() {
+    const currentUrl = window.location.pathname;
+
+    // If the current url is the root path, we don't want to show the deeplink prompt
+    if (currentUrl === '/') {
+        return false;
+    }
+
+    // Check if currentUrl pathname exactly matches any of the deny list routes
+    const existsOnDenyList = !_.includes(_.keys(CONST.DEEPLINK_PROMPT_DENYLIST), currentUrl);
```

Comment:
Is this intended? It's matching against keys, not values. Also, at least in Chrome, I see the window.location.pathname includes a slash at the beginning. (I think going with `useRoute` and react-navigation route names might be easiest)

Reply:
That's a good point. I compared on both Chrome and Firefox but attributed the change to the logic and not the browser, I'll re-evaluate this.